iron-overlay-behavior.html

```diff
@@ -167,10 +167,33 @@
       return this._focusedChild || Polymer.dom(this).querySelector('[autofocus]') || this;
     },
 
+    /**
+     * Nodes that can receive focus (overlay included). They will be ordered by their tabindex.
+     * @type [Node]
+     */
+    get _focusableNodes() {
+      var focusables = Polymer.dom(this).querySelectorAll('a[href]:not([tabindex="-1"]),area[href]:not([tabindex="-1"]),input:not([disabled]):not([tabindex="-1"]),select:not([disabled]):not([tabindex="-1"]),textarea:not([disabled]):not([tabindex="-1"]),button:not([disabled]):not([tabindex="-1"]),iframe:not([tabindex="-1"]),[tabindex]:not([tabindex="-1"]),[contentEditable=true]:not([tabindex="-1"])');
```

Comment:
It seems like this string could be documented a little more cleanly by splitting out the elements into groups (as arrays) and generating this nasty selector string with code. Example:

```javascript
var AUTOFOCUSABLE_WITH_DISABLED = [
  'a',
  'area',
  'input',
  // etc...
]

var AUTOFOCUSABLE_WITHOUT_DISABLED = [
  '[tabindex]',
  'iframe',
  '[contenteditable=true]',
  // etc..
];
```

Then, iterate over the lists to generate the selector at registration time. This will enable the scary selector to be somewhat self-documenting. WDYT?